src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const discountOne = 50;
+  const discountTwo = 20;
+  const cost = days * rentCost;
+
+  if (days < 3) {
+    return cost;
+  }
+
+  if (days < 7) {
+    return cost - discountTwo;
+  } else {
+    return cost - discountOne;
+  }
```

Comment:
else is redundant here, you can write only return
```suggestion
  }
  
  return cost - discountOne;
```